src/WorkerIndexing/WorkerIndexingHelper.cs

```diff
@@ -0,0 +1,51 @@
+﻿//
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+//
+
+using Microsoft.Azure.WebJobs.Script.Grpc.Messages;
+using Newtonsoft.Json;
+using System;
+using System.Collections.Generic;
+using System.Management.Automation;
+using System.Management.Automation.Runspaces;
+
+namespace Microsoft.Azure.Functions.PowerShellWorker.WorkerIndexing
+{
+    internal class WorkerIndexingHelper
+    {
+        const string GetFunctionsMetadataCmdletName = "Get-FunctionsMetadata";
```

Comment:
Please include the module name, e.g., `"AzureFunctions.PowerShell.SDK\\Get-FunctionsMetadata"`

Reply:
Done